services/web-app/pages/about-carbon/community-sites.mdx

```diff
@@ -0,0 +1,168 @@
+---
```

Comment:
Existing page: https://carbondesignsystem.com/community/domain-guidance

There's a redirect set.

Because of the image paths in this file, it couldn't be remote MDX?

Reply:
We're not being consistent in when we move MDX files into carbon-platform, vs carbon-platform branch on carbon-website repo, are we?

Reply:
got it!